### ISSUE TITLE
Document Daily Progress & New Tasks -- Casey

### ISSUE THREAD

**Issue by Ameobea** [Ameobea/network_research#28]
This is set as a daily task for each student Team Member.
Update the Asana project tasks, and research record.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/118867524991974/137797242216685)

**Comment by Ameobea** [comment]
I spend the majority of today importing and processing the new networks that I downloaded yesterday.  They were mostly in adjacency list format, which I had to implement in the processor code.  It took me a long time to figure out how to import that with networkx and after trying to roll my own solution by reading elements out of the list one by one and manually adding them to an empty network, I found a networkx function that did what I needed it to.  

After that was done, I created a HTML interface for use with the pair correlation scatter plots created by `correlate` and `plotCorrelations` in 7d46e3a06be620b74abbd7a518478cc8aa4422dc and a8e8de937ff91f13464a254e7f55a403e963bca1 which has been tested and works like I hoped it would.
